woocommerce/woocommerce-ios
# Update assets for free trial and prologue screens

Part of: #10374


## Description

This PR updates the asset used on the free trial summary screen.

The text and asset on the prologue screen has also been updated. The buttons' colors will need to be updated in a separate PR.

## Testing instructions

- Log out of the app, notice that the prologue screen displays the updated text.
- Select Create a New Store button and log in. Notice that the free trial summary screen has the updated assets.

## Screenshots

Free trial summary:

| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/ce837c55-bda4-4b71-9cc7-85c4ff927988" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/055c1904-806f-4874-98dc-1937960c3f7e" width=320 /> |

Prologue:
| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/d43da224-6771-4903-b8c0-5d4a76e5ada0" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/61bf7899-cccc-4eb9-a193-5ae5e28cf31c" width=320 /> | 

| iPhone SE | iPad 12.9" |
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/107759de-52ce-418b-a668-456ec6972c47" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/58a6a29c-ce09-4db6-ac4d-b00041c99230" width=320 /> |






---
